### DISCUSSION TITLE
Problems with importing Amber trajectory .nc with topology .prmtop

### DISCUSSION THREAD

**Discussion by vagra98** [BradyAJohnston/MolecularNodes#335] [General]
I'm trying to import a trajectory done in Amber with .nc file format and topology .prmtop but nothing shows up and no error is given. I did also tried to convert everything using Parmed to GROMACS type files but also without success. Any idea of what could be going wrong? I'm doing everything via the GUI in the MD Trajectory sector inside Scene. Thanks again Brady :)

**Comment by BradyAJohnston** [comment]
Resolved in #337
